pkg/daemon/ceph/config/config.go

```diff
@@ -209,14 +209,8 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 		currentMonPort := cephutil.GetPortFromEndpoint(monitor.Endpoint)
 
 		monPorts := [2]string{strconv.Itoa(int(Msgr2port)), strconv.Itoa(int(currentMonPort))}
-		msgr2Endpoint := net.JoinHostPort(monIP, monPorts[0])
 		msgr1Endpoint := net.JoinHostPort(monIP, monPorts[1])
-
-		if cluster.CephVersion.IsAtLeastNautilus() {
-			monHosts[i] = "[v2:" + msgr2Endpoint + ",v1:" + msgr1Endpoint + "]"
-		} else {
-			monHosts[i] = msgr1Endpoint
-		}
+		monHosts[i] = msgr1Endpoint
```

Comment:
Will the operator always use msgr1 now? Or do we update the client config to msgr2 somewhere else?

Reply:
At this point, it'll use v1 to communicate, I guess we can switch that back when we start deprecating Mimic.